src/UpdateTracker.js

```diff
@@ -41,14 +41,17 @@ export default class UpdateTracker {
   unsubscribe(...urls) {
     for (let url of urls) {
       url = url.replace(/#.*/, '');
-      if (url in subscribers)
+      if (url in subscribers) {
         subscribers[url].delete(this.subscriber);
+        delete subscribers[url];
```

Comment:
The above will cause issues when there are multiple `UpdateTracker` instances. The idea is that `subscribers[url]` is shared across _all_ `UpdateTracker` instances, representing a set of _all_ subscribed `UpdateTracker`s. When `unsubcribe` is on a _single_ instance, it should subscribe only that instance.

Reply:
Reverted this change.